tests/CMakeLists.txt

```diff
@@ -30,10 +30,24 @@ add_executable(test_status_enum
 target_link_libraries(test_status_enum rabbitmq-static)
 add_test(status_enum test_status_enum)
 
+add_executable(test_basic
+               test_basic.c)
+target_link_libraries(test_basic rabbitmq-static)
+# TODO actually, these tests should be executed, but ignored if no rabbitmq broker is running (which currently is the case on travis-ci Mac OS X builds)
+# However, the following does not work:
+#list(APPEND CTEST_CUSTOM_TESTS_IGNORE 
```

Comment:
`CTEST_CUSTOM_TESTS_IGNORE` requires writing a separate file `CTestsCustom.cmake`.  For now we can ignore that.  Delete this comment.

Reply:
Ok. Done